packages/loader/types/tsconfig.json

```diff
@@ -3,7 +3,6 @@
     "module": "commonjs",
     "lib": ["dom", "es6"],
     "strict": true,
-    "skipLibCheck": true,
```

Comment:
Stop omitting TypeScript errors from automated tests: `sed --in-place /skipLibCheck/d **/tsconfig.json`

Reply:
This option enabled makes ts ignoring errors in `.d.ts`, right? But what if there are some errors from `node_modules`?

Reply:
> This option enabled makes ts ignoring errors in `.d.ts`, right?

Yup.

> But what if there are some errors from `node_modules`?

If a way exists to verify our `.d.ts` without also checking declarations in `node_modules`, I don't know it ... :thinking: Could we use our dependencies/devDependencies and semvers to keep broken types out of `node_modules`? If that works, it would probably be the friendliest thing to our downstream types consumers?

Reply:
OK, maybe we can raise a feature request for TypeScript?

Reply:
I ran into an issue: Since dropping `skipLibChecks` we're checking the following `.d.ts`:
- our explicit `node_modules` dependencies
- every `node_modules/@types` declaration :joy:

https://github.com/mdx-js/mdx/pull/1622 add [`"types": []`](https://www.typescriptlang.org/tsconfig#types), to at least only check our explicit dependencies.